x-pack/plugins/security_solution/public/flyout/document_details/left/components/correlations_details_alerts_table.tsx

```diff
@@ -22,80 +26,50 @@ import { ExpandablePanel } from '../../../shared/components/expandable_panel';
 import { InvestigateInTimelineButton } from '../../../../common/components/event_details/table/investigate_in_timeline_button';
 import { ACTION_INVESTIGATE_IN_TIMELINE } from '../../../../detections/components/alerts_table/translations';
 import { getDataProvider } from '../../../../common/components/event_details/table/use_action_cell_data_provider';
+import { DocumentDetailsPreviewPanelKey } from '../../shared/constants/panel_keys';
+import { ALERT_PREVIEW_BANNER } from '../../preview';
 
 export const TIMESTAMP_DATE_FORMAT = 'MMM D, YYYY @ HH:mm:ss.SSS';
 const dataProviderLimit = 5;
 
-export const columns = [
```

Comment:
any reason for moving this constant inside the component? I think we should be able to keep it outside the component no?

Reply:
it is because of the feature flag `isPreviewEnabled`. I guess we could make it into a `getColumns` function..? we have both approach (keep it outside / inside) in the flyout code though.

I kept it inside and memoized the column variable, let me know if you feel strongly otherwise!